.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "dlib"]
-	path = thirdparty/dlib
-	url = https://github.com/stream-labs/dlib.git
 [submodule "freetype2"]
 	path = thirdparty/freetype2
 	url = https://github.com/stream-labs/freetype2.git
+[submodule "thirdparty/dlib"]
+	path = thirdparty/dlib
+	url = https://github.com/stream-labs/dlib
```

Comment:
Thanks for drawing attention to broken submodule. It was not about the url, I made a mistake when updating it. I fixed it now. See if you can update your submodules and have it work now and let me know if there is still an issue